scripts/xtensa-build-all.sh

```diff
@@ -287,19 +325,25 @@ do
 	( set -x # log the main commands and their parameters
 	cmake -DTOOLCHAIN="$TOOLCHAIN" \
 		-DROOT_DIR="$ROOT" \
+		"${MEU_OPENSSL_OPTION}" \
+		"${MEU_PATH_OPTION}" \
 		"${PRIVATE_KEY_OPTION}" \
 		..
 
 	make ${PLATFORM}${DEFCONFIG_PATCH}_defconfig
 	)
 
+	rm -fr override.config
```

Comment:
Nit: this can be replaced with `cp -f` below (and the `-r` doesn't really make sense)

Reply:
No wait: neither `rm` nor `-f` in `cp -f` should be needed at all, this is a brand new directory. If any is needed something has gone terribly wrong.